vue/src/Trigger/TriggerEdit.vue

```diff
@@ -655,6 +661,14 @@ export default defineComponent({
         translate('TagManager_CapabilityUseCustomTemplates'),
       );
     },
+    triggerInlineHelpText() {
+      const triggerInlineHelpLink = 'https://matomo.org/?post_type=faq&p=74424';
+      return translate(
+        'TagManager_TriggerConditionsHelpText',
+        `<a href="${triggerInlineHelpLink}" target="_blank" rel="noreferrer noopener">`,
```

Comment:
Should this be using the `externalLink` function to automatically add the campaign parameters to the URL?

Reply:
Good catch @snake14 :+1:  I almost forgot about it, @araichyk FYI its a way to link the FAQs with more info to track from where the traffic is coming, it was done for the marketing team.
eg `https://matomo.org/faq/tag-manager/create-a-trigger-to-track-interactions-on-all-nested-elements/?mtm_campaign=Matomo_App&mtm_source=Matomo_App_OnPremise&mtm_medium=App.TagManager.manageTriggers`

Reply:
Done